zkstack_cli/crates/zkstack/src/commands/dev/commands/lint.rs

```diff
@@ -133,3 +149,44 @@ fn lint_contracts(shell: &Shell, ecosystem: &EcosystemConfig, check: bool) -> an
 
     Ok(())
 }
+
+fn lint_autocompletion_files(_shell: &Shell, check: bool) -> anyhow::Result<()> {
+    let completion_folder = Path::new("./zkstack_cli/crates/zkstack/completion/");
+    if !completion_folder.exists() {
+        logger::info("WARNING: Please run this command from the project's root folder");
+        return Ok(());
+    }
+
+    // Array of supported shells
+    let shells = ["bash", "zsh"];
+
+    for shell in &shells {
+        let tmp_path = completion_folder.join(format!("_tmp_zkstack_{}", shell));
+        let tmp_file = File::create(tmp_path.clone()).context("Failed to create file")?;
+        let mut writer = BufWriter::new(tmp_file);
```

Comment:
why do you need temp file? 
You can simply write to Vec<u8> and compare it later

Reply:
Nice! https://github.com/matter-labs/zksync-era/pull/3097/commits/fa44791ca3a34b6cb29471eaa1cb0d3b89e88d5d